### ISSUE TITLE
mySQL status on Home page

### ISSUE THREAD

**Issue by Jmos** [crazy-max/neard#134]
Incorrect mysql running status homepage sugest stopped but running

Details
![mysql](https://cloud.githubusercontent.com/assets/5389792/14878845/5c3759a2-0d26-11e6-8c92-d6bb5fa1a2af.png)
Verbose [Log](https://www.sendspace.com/file/4hj82a)

[Core temp](https://www.sendspace.com/file/bs2yfz)

win7x64 Neard  1.0.18

**Comment by crazy-max** [comment]
There is something in the error log :

```
Access denied for user 'root'@'localhost' (using password: NO) in /core/classes/bins/class.bin.mysql.php on line 199
```

Do you changed mysql root password ?

**Comment by Jmos** [comment]
Yes i changed because a development purpose I need change some were else

On Thu, Apr 28, 2016 at 10:25 AM, Cr@zy <notifications@github.com> wrote:

> There is something in the error log :
>
> Access denied for user 'root'@'localhost' (using password: NO) in /core/classes/bins/class.bin.mysql.php on line 199
>
> Do you changed mysql root password ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/crazy-max/neard/issues/134#issuecomment-215350404>
>



-- 
-------------------------------------------
mailto:janos.meggyeshazi@gmail.com
+36 20 776 9514

**Comment by crazy-max** [comment]
Hum ok i will make an improvement on Neard to be able to change the MySQL / MariaDB root password

**Comment by Jmos** [comment]
Tanx


On Thu, Apr 28, 2016 at 1:44 PM, Cr@zy <notifications@github.com> wrote:

> Hum ok i will make an improvement on Neard to be able to change the MySQL
> / MariaDB root password
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/crazy-max/neard/issues/134#issuecomment-215399055>
>



-- 
-------------------------------------------
mailto:janos.meggyeshazi@gmail.com
+36 20 776 9514

**Comment by crazy-max** [comment]
Solved on Neard 1.0.19

You will have to change the root password in MySQL and/or MariaDB menu :
* Left click on Neard tray icon
* MySQL/MariaDB > Service > Change root password
